stackdriver/README.md

```diff
@@ -0,0 +1,61 @@
+# stackdriver
+
+This gem is a convenience package for loading all Stackdriver gems in the google-cloud-ruby project. 
+- [google-cloud-logging](../google-cloud-logging)
+- [google-cloud-error_reporting](../google-cloud-error_reporting)
+- [google-cloud-monitoring](../google-cloud-monitoring)
+
+Please see the top-level project [README](../README.md) for more information about the individual Stackdriver google-cloud-ruby gems.
+
+## Quick Start
+
+```sh
+$ gem install stackdriver
+```
+
+## Overview
```

Comment:
I had a save concern, but I thought it would be better to show user the before and after contrast. I can remove the before example code.